Source/Core/Core/IOS/Network/Net.cpp

```diff
@@ -625,9 +625,7 @@ IPCCommandResult NetIPTop::IOCtl(const IOCtlRequest& request)
     // Do the level/optname translation
     int nat_level = -1, nat_optname = -1;
 
-    for (auto& map : opt_level_mapping)
-      if (level == map[1])
-        nat_level = map[0];
+    nat_level = opt_level_mapping[level];
```

Comment:
Shouldn't you handle the case where `opt_level_mapping` doesn't contain `level`?